rental.rb

```diff
@@ -0,0 +1,11 @@
+class Rental
+  attr_accessor :date, :book, :person
+
+  def initialize(date, book, person)
+    @date = date
+    @book = book
+    @person = person
+    book.add_rental(self)
+    person.add_rental(self)
```

Comment:
- [ ] We have set up `attr_accessor` for rentals in the `Book` and `Person` classes. Now is the time to use it. We can use the accessor to add the rental to the Book's and Person's rentals array. That way we are achieving the `belongs-to` side of the relationship. 💯️

```
  @book = book
  book.rentals << self
  @person = person
  person.rentals << self
```

Reply:
Thanks! the changes has been made